CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## Version 1.12.1 (Under active development)
 
+### AppCenter
+
+#### Android
+
+* **[Feature]** AppCenter SDK now supports the User ID string, with a maximum of 256 characters, that applies to crashes, handled errors, and push logs. Settable via `AppCenter.SetUserId()`.
```

Comment:
This should include a not like in the Android and iOS SDKs that this is not supported in that backend yet.

See the first bullet here: https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/CHANGELOG.md#version-1120